src/lambda_test_settings.py

```diff
@@ -0,0 +1,87 @@
+import os
+import json
+
+from constants import SettingFileNames
+
+from s3_manager import S3Manager
+from settings.settings_reader import (
+    MonitoringSettingsReader,
+    RecipientsSettingsReader,
+)
+
+
+def lambda_handler(event, context):
+    # Get environment variables
```

Comment:
Please add a doc-string for this function.

Reply:
Added